tools/fix-ext.mjs

```diff
@@ -6,11 +6,15 @@ import { globby } from 'globby';
 // manually rename the common.js files and their imports to .js
 const mjsFiles = await globby(['dist/cjs/**/*.mjs', '!**/node_modules']);
 mjsFiles.forEach(async (pathname) => {
-  // 1. Rename imports
+  // 1. Rename imports & exports
   const fileContent = await fs.readFile(pathname, 'utf8');
   await fs.writeFile(
     pathname,
-    fileContent.replaceAll(/require\(["']([^"']*)\.mjs["']\)/g, "require('$1.js')"),
+    fileContent
+      // from `.mjs` to `.js`
+      .replaceAll(/require\(["']([^"']*)\.mjs["']\)/g, "require('$1.js')")
+      // From export.default to module.exports
+      .replaceAll('exports.default', 'module.exports'),
```

Comment:
Note to self: hacked around this, but not the right approach. The other modules don't do this. I believe it's happening because `inquirer` mixes default and named exports. Might want to clean that instead